frontend/src/bundles/studio/components/timeline/row/avatars-row.tsx

```diff
@@ -0,0 +1,39 @@
+import { Text } from '~/bundles/common/components/components.js';
+import { RowNames } from '~/bundles/studio/enums/enums.js';
+import { type TimelineItemWithSpan } from '~/bundles/studio/types/types.js';
+
+import { Item } from '../item/item.js';
+import { Row } from './row.js';
+
+type Properties = {
+    items: Array<TimelineItemWithSpan>;
+    destinationPointerValue?: number | undefined;
```

Comment:
```suggestion
    destinationPointerValue?: number;
```
Please, change it everywhere

Reply:
I reviewed my components once again and noticed that `destinationPointer` in most of the cases is not optional, so I changed it to `destinationPointerValue: number | null`